products/memgraph/upcoming/reference_guide/configuration.md

```diff
@@ -56,6 +56,7 @@ Each configuration setting is in the form: `--setting-name=value`.
 | --also-log-to-stderr=false | log messages go to stderr in addition to logfiles | `[bool]` |
 | --data-directory=/var/lib/memgraph | Path to directory in which to save all permanent data. | `[string]` |
 | --log-file=/var/log/memgraph/memgraph.log | Path to where the log should be stored. | `[string]` |
+| --log-level | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL | `[string]` |
```

Comment:
@g-despot Is this generated or do we manually add this?

Reply:
Still being done manually.